tools/cli.js

```diff
@@ -24,14 +24,64 @@ const log = (msg) => {
   if (userInput) rl.write(userInput);
 };
 
+function completeByFields(input, object) {
+  return Object.keys(object)
+    .filter(c => !c.startsWith('_') && (!input || c.startsWith(input)));
+}
+
+function tryComplete(input, completer) {
+  const completions = completer._complete([input], 0)[0];
```

Comment:
@aqrln hm, at first I thought to always use fields for completion, but now I see that it'd to be better to state the completions manually, then i can get rid of the underscore.